src/geobr.jl

```diff
@@ -0,0 +1,639 @@
+amazon(; year=2012, kwargs...) = get("amazonia_legal", year; kwargs...)
+
+"""
+    biomes(; year=2019, kwargs...)
+
+Get biomes data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2019).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Biomes data for the specified year.
+"""
+biomes(; year=2019, kwargs...) = get("biomes", year; kwargs...)
+
+"""
+    disasterriskarea(; year=2010, kwargs...)
+
+Get disaster risk area data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2010).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Disaster risk area data for the specified year.
+"""
+disasterriskarea(; year=2010, kwargs...) = get("disaster_risk_area", year; kwargs...)
+
+"""
+    healthfacilities(; year=2013, kwargs...)
+
+Get health facilities data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2013).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Health facilities data for the specified year.
+"""
+healthfacilities(; year=2013, kwargs...) = get("health_facilities", year; kwargs...)
+
+"""
+    indigenousland(; date=201907, kwargs...)
+
+Get indigenous land data for a given date.
+
+Arguments:
+- `date`: Date of the data (default: 201907).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Indigenous land data for the specified date.
+"""
+indigenousland(; date=201907, kwargs...) = get("indigenous_land", date; kwargs...)
+
+"""
+    metroarea(; year=2018, kwargs...)
+
+Get metropolitan area data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2018).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Metropolitan area data for the specified year.
+"""
+metroarea(; year=2018, kwargs...) = get("metropolitan_area", year; kwargs...)
+
+"""
+    neighborhood(; year=2010, kwargs...)
+
+Get neighborhood data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2010).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Neighborhood data for the specified year.
+"""
+neighborhood(; year=2010, kwargs...) = get("neighborhood", year; kwargs...)
+
+"""
+    urbanarea(; year=2015, kwargs...)
+
+Get urban area data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2015).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Urban area data for the specified year.
+"""
+urbanarea(; year=2015, kwargs...) = get("urban_area", year; kwargs...)
+
+"""
+    weightingarea(weighting; year=2010, kwargs...)
+
+Get weighting area data for a given year.
+
+Arguments:
+- `weighting`: Weighting area code or abbreviation.
+- `year`: Year of the data (default: 2010).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Weighting area data for the specified year.
+"""
+function weightingarea(weighting; year=2010, kwargs...)
+    code = isa(weighting, Number) ? weighting : nothing
+    abbrev = isa(weighting, AbstractString) ? weighting : nothing
+    get("weighting_area", year, code, abbrev, kwargs...)
```

Comment:
I think I am following the logic now. I understand that the user can either specify a numerical code or string abbreviation. Based on any of the two options you download the data.

Please refactor it for improved readability:

```julia
if weighting isa Number
  get("weighting_area", year; code=weighting, kwargs...)
elseif weighting isa AbstractString
  get("weighting_area", year; abbrev=weighting, kwargs...)
end
```

Even better would be to refactor the `get` function to only accept a single option `code`, which can be either a number or a string. The logic would live inside the `get` function. I understand that users cannot specify code + abbrev at the same time, these are just alternative specs.